packages/toolkit/src/view/recipe-editor/commands/ComponentCmdo.tsx

```diff
@@ -777,14 +782,18 @@ export const ComponentCmdo = () => {
               </ScrollArea.Root>
             </div>
             <Separator orientation="vertical" className="!mx-3" />
-            <div className="flex flex-col  w-full h-full">
-              {selectedComponentDefinition &&
-              isComponentDefinition(selectedComponentDefinition) ? (
-                <ScrollArea.Root className="flex shrink-0 mb-auto h-[280px]">
-                  <CommandGroup headingWrapperClassName="px-2" heading="Task">
+            <div className="w-full h-full relative">
+              <div className="absolute top-0 left-0 w-full h-full overflow-hidden">
+                {selectedComponentDefinition &&
+                isComponentDefinition(selectedComponentDefinition) ? (
+                  <ScrollArea.Root className="flex shrink-0 mb-auto h-full [&>*>*>*:last-child]:mb-8">
```

Comment:
@orangecoloured Why we need to use relative and absolute here?

Reply:
Because all the parents use height 100% and the list expands too much without a set height. It breaks the layout. So by using an absolute inside the relative I limit its height.